domains/auth/Auth.API/API/Controllers/AuthController.cs

```diff
@@ -33,4 +34,17 @@ public AuthController(ILogger<AuthController> logger, IOidcProviders oidcProvide
         return oidcProviders.CreateAuthorizationUri(state);
     }
 
+
+    [HttpPost]
+    [Route("/invalidate")]
+    public IActionResult Invalidate([Required] AuthState state)
+    {
+        if (state.IdToken == null)
+        {
+            return BadRequest();
```

Comment:
No information about what is wrong in the request is returned. Is there a reason for this?

For other domains (emission and measurement) we are using FluentValidation to validate the incoming request. FluentValidation also takes care of generating validation messages. You can give it a look, but maybe in this case we have to be a bit careful about using FluentValidation as (and this is how I understand it) AuthState will be filled out during the onboarding flow. This mean that the validation performed is going to be specific for each endpoint.

Reply:
Let's just return a text response - this is an improvement over existing auth service